src/gui/core/plugin/userui/src/main/java/org/geoserver/geofence/gui/server/service/impl/ProfilesManagerServiceImpl.java

```diff
@@ -22,6 +13,14 @@
 import java.util.SortedSet;
 import java.util.TreeSet;
 
+import org.geoserver.geofence.gui.client.ApplicationException;
+import org.geoserver.geofence.gui.client.model.UserGroup;
+import org.geoserver.geofence.gui.client.model.data.ProfileCustomProps;
+import org.geoserver.geofence.gui.client.model.data.rpc.RpcPageLoadResult;
+import org.geoserver.geofence.gui.server.service.IProfilesManagerService;
+import org.geoserver.geofence.gui.service.GeofenceRemoteService;
+import org.geoserver.geofence.services.dto.ShortGroup;
+import org.geoserver.geofence.services.exception.NotFoundServiceEx;
```

Comment:
Please avoid committing packages cleanup if this is the only change inside a class